modules/ossm-config-otel.adoc

```diff
@@ -99,41 +99,51 @@ spec:
 Once you verify that you can see traces, lower the `randomSamplingPercentage` value or set it to `default` to reduce the number of requests.
 ====
 
-. Deploy the `bookinfo` application in `bookinfo` namespace:
+. Create the `bookinfo` namespace by running the following command:
 +
 [source, terminal]
 ----
 $ oc create ns bookinfo
 ----
+
+. Depending on the update strategy you're using, enable sidecar injection in the namespace by running the appropriate command(s):
+
+.. If you are using the `InPlace` update strategy and your `Istio` resource is named `default`, then run the following command:
 +
 [source, terminal]
 ----
-$ oc label ns <namespace_name> istio.io/rev= # <1>
+$ oc label ns bookinfo istio-injection=enabled
 ----
 +
+.. If you are using the `RevisionBased` update strategy, then run the following commands:
++
+... Find your `<revision-name>` by running the following command:
++
 [source, terminal]
 ----
-$ oc apply -f https://raw.githubusercontent.com/istio/istio/release-1.23/samples/bookinfo/platform/kube/bookinfo.yaml -n bookinfo
+$ oc get istiorevisions.sailoperator.io
 ----
-<1> If you named your Istio resource `default` and are using the `InPlace` upgrade strategy, use `oc label ns bookinfo istio-injection=enabled`.
 +
-[NOTE]
-====
-To find your `<revision-name>`, run the following command:
-
+.Example output:
+[source,terminal]
+----
+NAME              TYPE    READY   STATUS    IN USE   VERSION   AGE
+default-v1-23-0   Local   True    Healthy   True    v1.23.0   3m33s
+----
++
+... Run the following command to label the namespace with the revision name to enable sidecar injection:
```

Comment:
A few thoughts on this step. For one, I guess it's not necessary, but reversing the order of the phrasing so that it's worded like "take the action by running the following command" would make it more consistent. Additionally, and I think this is probably why it's worded the current way, i think you could remove the "to enable sidecar injection" phrase because that wording is already introduced in the step this is nested under.

Here's what I would suggest:
```suggestion
... Label the namespace with the revision name by running the following command:
```